tools/cli/pkg/command/upgrade_cluster.go

```diff
@@ -57,5 +59,13 @@ func (cmd *UpgradeClusterCommand) Run() error {
 		return errors.Wrap(err, "while triggering kyma upgrade")
 	}
 	fmt.Println("OrchestrationID:", ur.OrchestrationID)
+
+	if !cmd.orchestrationParams.DryRun {
+		slack_title := `upgrade cluster`
+		slack_err := SendSlackNotification(slack_title, cmd.cobraCmd, "OrchestrationID:"+ur.OrchestrationID)
```

Comment:
Calling this function always try to send the slack notification, even though the API URL is not configured/empty

Reply:
hi @ebensom you are right and i modified as below and could u help to review again? many thanks.
`if !cmd.orchestrationParams.DryRun && GlobalOpts.SlackAPIURL() != "" {`